### DISCUSSION TITLE
Photo dimensions for Cover Photo

### DISCUSSION THREAD

**Discussion by cloud303-mherbert** [outline/outline#4950] [General] [answered]
Can someone let me know the dimensions a photo should be for the cover photo and a doc?

**Comment by tommoor** [comment, marked as answer]
What cover photo are you referring to?

**Reply by cloud303-mherbert** [reply]
@tommoor Hello Tom,

Sorry for not being more specific. When you create a new doc, at the top you can upload a photo for a banner. I'd like to know the recommended sizing for that photo.

![image](https://user-images.githubusercontent.com/122323161/221951328-5dcf6848-e1c4-4fe3-bc74-bd35dbccff69.png)

**Reply by tommoor** [reply]
Where are you accessing Outline? This functionality does not exist in the main codebase

**Reply by cloud303-mherbert** [reply]
ah thanks for pointing that out!

**Reply by tommoor** [reply]
Has your install been customized?

**Reply by cloud303-mherbert** [reply]
yea, Ill check our code again.
